whats-new-panel.yaml

```diff
@@ -113,22 +35,59 @@
   targeting: firefoxVersion >= 78
   trigger:
     id: whatsNewPanelOpened
-- id: WHATS_NEW_BADGE_78
+- id: WHATS_NEW_BADGE_80
   content:
     action:
       id: show-whatsnew-button
     badgeDescription:
       string_id: cfr-badge-reader-label-newfeature
-    bucket_id: WHATS_NEW_BADGE_78
```

Comment:
Can you also copy `WHATS_NEW_BADGE_78` to `archive/whats-new-panel-archived.yaml`?